qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/CreateCustomerView.groovy

```diff
@@ -73,29 +73,41 @@ class CreateCustomerView extends FormLayout {
 
         this.affiliationComboBox = generateAffiliationSelector(sharedViewModel.affiliations)
         affiliationComboBox.setRequiredIndicatorVisible(true)
+        this.affiliationButton = new Button(VaadinIcons.PLUS)
```

Comment:
I like this button 👍

Reply:
> What's the reason to rearrange the elements?
I wanted the `PLUS` button and the affiliation selection in one row. This was not possible with Vaadin 8 FormLayouts